Workspaces/MiniWorldWorkspace/MiniWorldWorkspace.cs

```diff
@@ -106,9 +105,10 @@ public float zoomSliderMax
 
 		public override void Setup()
 		{
+			const float yBounds = 0.5f;
 			// Initial bounds must be set before the base.Setup() is called
-			minBounds = new Vector3(MinBounds.x, MinBounds.y, 0.25f);
-			m_CustomStartingBounds = new Vector3(MinBounds.x, MinBounds.y, 0.5f);
+			minBounds = new Vector3(MinBounds.x, yBounds, 0.25f);
```

Comment:
This was to increase the height of the chessboard. It's using the default minbounds, but we want the height to be a little bigger, hence the new const.